src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -267,7 +267,7 @@ def __init__(self, config):
             self.proxy = self.setupMyProxy()
 
         # Build a request string
-        self.reqStr = "(Memory >= 1 && OpSys == \"LINUX\" ) && (Arch == \"INTEL\" || Arch == \"X86_64\") && stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) && ((REQUIRED_OS==\"any\") || (GLIDEIN_REQUIRED_OS==REQUIRED_OS))"
```

Comment:
Hmm, I'd expect any pilots to provide at least 1MB of memory, any motivation to explicitly ask for it?